plugin/src/main/java/com/h3xstream/findsecbugs/android/BroadcastDetector.java

```diff
@@ -37,11 +37,14 @@ public void sawOpcode(int seen) {
         //printOpCode(seen);
 
         if (seen == Constants.INVOKEVIRTUAL &&
-                (getNameConstantOperand().equals("sendBroadcast") ||
-                getNameConstantOperand().equals("sendBroadcastAsUser") ||
-                getNameConstantOperand().equals("sendOrderedBroadcast") ||
-                getNameConstantOperand().equals("sendOrderedBroadcastAsUser")
-                )) {
+                (
+                    getNameConstantOperand().equals("sendBroadcast") ||
+                    getNameConstantOperand().equals("sendBroadcastAsUser") ||
+                    getNameConstantOperand().equals("sendOrderedBroadcast") ||
+                    getNameConstantOperand().equals("sendOrderedBroadcastAsUser")
+                )
+                && !getClassConstantOperand().endsWith("LocalBroadcastManager")     // The LocalBroadcastManager object is safe. The broadcast doesn't leave the application scope.
```

Comment:
I would use `InterfaceUtils.isSubtype()` has a main method to confirm that it is a LocalBroadcastManager.
The check `!getClassConstantOperand().endsWith("LocalBroadcastManager")`  can still be present. It will be useful to have fallback when the complete class hierarchy can be loaded (aka missing classes during the scan).